spec/unit/widgets/multilinebox_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'rspec'
+
+module LibyuiClient
+  module Widgets
+    RSpec.describe Multilinebox do
+      let(:widget_controller) { instance_double('widget_controller') }
+      let(:filter) { FilterExtractor.new({ id: 'test_multiline' }) }
+      subject { Multilinebox.new(widget_controller, filter) }
+
+      describe '#max_length' do
+        let(:response) { double('Response', { body: [{ input_max_length: -1 }] }) }
+        it 'returns input max length' do
+          expect(widget_controller).to receive(:find).and_return(response)
+          expect(subject.max_length).to eq(-1)
+        end
+      end
+
+      describe '#set' do
+        it 'sets text' do
+          expect(widget_controller).to receive(:send_action)
+          expect(subject.set('text
+text in new line')).to equal(subject)
```

Comment:
doesn't work with double quotes and `\n` to set it in one line?
I guess similar what I mentioned in the comment above.

Reply:
Done